acceptance/tests/vault/vault_test.go

```diff
@@ -48,16 +64,15 @@ func TestVault_BootstrapConsulGossipEncryptionKey(t *testing.T) {
 	vaultClient := vaultCluster.VaultClient(t)
 
 	// Create the Vault Policy for the gossip key.
-	logger.Log(t, "Creating the gossip policy")
-	rules := `
-path "consul/data/secret/gossip" {
-  capabilities = ["read"]
-}`
-	err := vaultClient.Sys().PutPolicy("consul-gossip", rules)
+	logger.Log(t, "Creating policies")
+	err := vaultClient.Sys().PutPolicy("consul-gossip", gossipPolicy)
+	require.NoError(t, err)
+
+	err = vaultClient.Sys().PutPolicy("connect-ca", connectCAPolicy)
 	require.NoError(t, err)
 
-	// Create the Auth Roles for consul-server + consul-client.
-	logger.Log(t, "Creating the consul-server and consul-client-roles")
+	// Create the Auth Roles for consul-server and consul-client.
```

Comment:
Could we say "Create the Auth Roles for consul-server and consul-client which enables ___"? I'm not sure what auth roles do so maybe it'd be nice to add a tiny summary in the comment?

Reply:
Yeah, great call-out! I've added more info to this comment. Let me know if that makes sense!